### ISSUE TITLE
PDF link broken

### ISSUE THREAD

**Issue by dacoex** [matplotlib/devdocs#2]
these links are dead:
http://matplotlib.org/2.0.0b4/contents.html 
to
http://matplotlib.org/2.0.0b4/Matplotlib.pdf

&
http://matplotlib.org/devdocs/contents.html
to
http://matplotlib.org/devdocs/Matplotlib.pdf

**Comment by tacaswell** [comment]
Do you really use the pdf version of the docs?

**Comment by jenshnielsen** [comment]
We should probably add logic to only include that link to the pdf version if the pdf was build.

**Comment by dacoex** [comment]
@tacaswell 
Yes, for offline reading and it's also convenient to have the mpl docs open on PDF in a separate window while doing something else on browser like SO browsing or notebooks.
Thanks.

**Comment by tacaswell** [comment]
Would you be happy with the ability to download a zip-file of the HTML docs?

**Comment by dacoex** [comment]
the PDF was a nicely compact document. But since I know how difficult it is to create it, ZIP should do fine.

**Comment by tacaswell** [comment]
Last time I tried to build the pdf docs I got LaTeX exceptions, any help debugging that would be appreciated.

**Comment by jkseppan** [comment]
I was just able to build the pdf documentation on a Debian jessie host with a lot of TeX packages installed. The resulting file is temporarily [here](http://jks.iki.fi/tmp/Matplotlib.pdf). This is a large file:

    Output written on Matplotlib.pdf (2767 pages, 19299663 bytes).

If we want to offer a pdf version of the documentation, we should probably split this up into several files. There are various formatting problems, starting on the title page.
